nodeEditor/src/graphEditor.tsx

```diff
@@ -157,6 +158,18 @@ export class GraphEditor extends React.Component<IGraphEditorProps, IGraphEditor
             }
         });
 
+        this.props.globalState.onImportFrameObservable.add((source: any) => {
+            let editorData = this.props.globalState.nodeMaterial.editorData;        
+    
+            if (editorData instanceof Array) {

```

Comment:
is it required?

Reply:
I am not 100% sure. I added it because it was in GraphEditor.build() and since nodeMaterial.editorData is of type: any I figured it was better safe than sorry to add the check. I can remove it if you think it is not needed.

Reply:
Yeah I think it is safe to remove it